### DISCUSSION TITLE
Formily 官网（ https://formilyjs.org/ ）打开后多了一层 iframe

### DISCUSSION THREAD

**Discussion by zjffun** [alibaba/formily#2521] [Q&A 2.x] [answered]
我这里 Formily 官网（ https://formilyjs.org/ ）打开后多了一层 iframe ，已清缓存，浏览器是 Google Chrome Version 96.0.4664.55 (Official Build) (arm64)

访问 https://v2.formilyjs.org/ 正常。

![image](https://user-images.githubusercontent.com/25266120/143547076-f04fc152-f682-46be-93d8-411e7694a948.png)

**Comment by janryWang** [comment, marked as answer]
application里unregister service worker

**Comment by zjffun** [comment]
确实好了👍。之前一直当 service worker 有效期只有一天，看起来不是
![image](https://user-images.githubusercontent.com/25266120/143553546-28df3eea-5b57-4120-9626-e85af7ac90d1.png)
